test/fp_to_int_intrinsic.ll

```diff
@@ -0,0 +1,236 @@
+;; Ensure @llvm.fptosi.sat.* and @llvm.fptoui.sat.* intrinsics are translated
+
+; RUN: llvm-as %s -o %t.bc
+; RUN: llvm-spirv %t.bc -spirv-text -o - | FileCheck %s --check-prefix=CHECK-SPIRV
+; RUN: llvm-spirv %t.bc -o %t.spv
+; RUN: llvm-spirv -r %t.spv -o %t.rev.bc
+; RUN: llvm-dis < %t.rev.bc | FileCheck %s --check-prefix=CHECK-LLVM
+
+target datalayout = "e-i64:64-v16:16-v24:32-v32:32-v48:64-v96:128-v192:256-v256:256-v512:512-v1024:1024-n8:16:32:64"
+target triple = "spir64-unknown-unknown"
+
+
+; CHECK-SPIRV: Decorate [[SAT1:[0-9]+]] SaturatedConversion
```

Comment:
I think one variable SAT should be sufficient and can be reused in the remaining lines (14-28).

Thanks

Reply:
Multiple SAT variables are needed because llvm-spirv defines multiple SaturatedConversion IDs.  These are used to annotate ConvertFToS insns -- there is no single SaturatedConversion ID that can be checked with the multiple ConvertFToS insns.  **It would make this testcase cleaner if llvm-spirv could reuse a single SaturatedConversion ID**.